screenpy_selenium/resolutions/is_clickable.py

```diff
@@ -15,5 +16,9 @@ class IsClickable(BaseResolution):
         the_actor.should(See.the(Element(LOGIN_BUTTON), IsClickable()))
     """
 
+    matcher: IsClickableElement
     line = "clickable"
     matcher_function = is_clickable_element
+
+    def __init__(self) -> None:  # pylint: disable=useless-super-delegation
```

Comment:
Yeah, like this! This is exactly what i was talkin' about on https://github.com/ScreenPyHQ/screenpy/pull/12.